src/some-package/java/com/mainpackage/FileSyntaxError.java

```diff
@@ -0,0 +1,37 @@
+package com.mainpackage;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.S3Object;
+import java.io.IOException;
+import java.io.InputStream;
+import com.amazonaws.util.IOUtils;
+
+/**
+ * Even though this file contains weak crypto issues, CodeGuru Reviewer will not
+ * report any issues in it, because it has been excluded in aws-codeguru-reviewer.yml.
+ * 
+ * For more information, see the Amazon CodeGuru Reviewer User Guide:
+ * https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/welcome.html.
+ */
+public class FileSyntaxError {
+
+    // {fact rule=file-syntax-error@v1.0 defects=1}
```

Comment:
Are we using this meta-data in this repo as well, or is it only for the detector library?

Reply:
This metadata is from detector library. Let me remove this